deploy/jbossas/modeshape-jbossas-subsystem/src/main/java/org/modeshape/jboss/subsystem/ModeShapeSubsystemXMLReader_3_0.java

```diff
@@ -31,7 +31,7 @@
 import org.jboss.staxmapper.XMLElementReader;
 import org.jboss.staxmapper.XMLExtendedStreamReader;
 
-public class ModeShapeSubsystemXMLReader_2_1 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
+public class ModeShapeSubsystemXMLReader_3_0 implements XMLStreamConstants, XMLElementReader<List<ModelNode>> {
```

Comment:
not that i'm aware of, no.

For comparison, https://issues.jboss.org/browse/MODE-2454 changed the version from `2.0` to `2.1` between 2 `4.x` _minor_ releases. This is for a major release and is exactly the same thing....